houston/model/build.js

```diff
@@ -145,12 +145,7 @@ schema.methods.doBuild = function () {
     tag: this.cycle.tag,
     version: this.cycle.version
   })
-  .then(() => {
-    log.debug(`Building ${this.cycle.name} for ${this.arch} on ${this.dist}`)
```

Comment:
Why did you remove debug logging here? It could be useful in the future for developers to debug issues